packages/build/src/telemetry/main.js

```diff
@@ -20,40 +24,30 @@ const trackBuildComplete = async function ({
   durationNs,
   siteInfo,
   telemetry,
-  testOpts,
+  testOpts: { telemetryOrigin } = {},
```

Comment:
```suggestion
  testOpts: { telemetryOrigin },
```

I believe `normalize_flags.js` should assign a default value for `testOpts`?

Reply:
Addressed via https://github.com/netlify/build/pull/2438/commits/65310c4ca1ced732ea1f918be17e135dcd691c4d